src/main/cli/settings.c

```diff
@@ -955,6 +955,7 @@ const clivalue_t valueTable[] = {
     { "rpm_limit_i",                VAR_UINT16 |  MASTER_VALUE,  .config.minmaxUnsigned = { 0, 1000 },       PG_MIXER_CONFIG, offsetof(mixerConfig_t, rpm_limit_i) },
     { "rpm_limit_d",                VAR_UINT16 |  MASTER_VALUE,  .config.minmaxUnsigned = { 0, 100 },        PG_MIXER_CONFIG, offsetof(mixerConfig_t, rpm_limit_d) },
     { "rpm_limit_value",            VAR_UINT16 |  MASTER_VALUE,  .config.minmaxUnsigned = { 1, UINT16_MAX }, PG_MIXER_CONFIG, offsetof(mixerConfig_t, rpm_limit_value) },
+    { "rpm_limit_rpm_filter_cutoff",    VAR_UINT16 |  MASTER_VALUE,  .config.minmaxUnsigned = { 1, 1000 },        PG_MIXER_CONFIG, offsetof(mixerConfig_t, rpm_limit_rpm_filter_cutoff) },
```

Comment:
```suggestion
    { "rpm_limit_cutoff_hz",        VAR_UINT16 |  MASTER_VALUE,  .config.minmaxUnsigned = { 1, 1000 },        PG_MIXER_CONFIG, offsetof(mixerConfig_t, rpm_limit_rpm_filter_cutoff) },
```

Reply:
I'm worried "rpm_limit_cutoff_hz" may also be misleading as there are more than one filter used in the RPM limiter. The prefix "rpm_limit_" isn't enough to distinguish which filter cutoff hz we are adjusting. Perhaps "rpm_limit_AvgRPM_cutoff_hz" would be better, and not as easily confused with the gyro RPM Filter?

Also just a side note, I'm currently testing the addition of another filter to the RPM limiter D term. I've been calling that one "rpm_limit_d_term_filter_cutoff".